install_config/aggregate_logging.adoc

```diff
@@ -12,31 +12,33 @@ toc::[]
 
 == Overview
 
-OpenShift cluster administrators can aggregate and view logs for a range of
-OpenShift services using the EFK stack, which is a modified version of the
-https://www.elastic.co/videos/introduction-to-the-elk-stack[ELK stack]. This
-topic shows you how to deploy the stack using a template.
+OpenShift cluster administrators can aggregate logs and
+enable users to view them for a range of OpenShift services
+using the EFK stack, which is a modified version of the
+https://www.elastic.co/videos/introduction-to-the-elk-stack[ELK
+stack]. The EFK stack can be useful for viewing logs aggregated from
+hosts and applications, whether coming from multiple containers or even
+deleted pods.
 
 Three components make up the EFK logging stack:
 
-* Elasticsearch: an object store where all logs are fed.
-* Fluentd: gathers logs from nodes and feeds them to Elasticsearch.
-* Kibana: a web UI for Elasticsearch.
+* https://www.elastic.co/products/elasticsearch[Elasticsearch]: an object store where all logs are stored.
+* http://www.fluentd.org/architecture[Fluentd]: gathers logs from nodes and feeds them to Elasticsearch.
+* https://www.elastic.co/guide/en/kibana/current/introduction.html[Kibana]: a web UI for Elasticsearch.
 
-The EFK stack can be useful for tracking logs and analytics to help see how
-application users are using your application. For example, you can use
-https://www.elastic.co/products/elasticsearch[Elasticsearch] to store logs and
-transaction data, use http://www.fluentd.org/architecture[Fluentd] to collect
-the data, and use
-https://www.elastic.co/guide/en/kibana/current/introduction.html[Kibana] to
-filter and examine the data on the Kibana console.
+Once deployed on a cluster, the stack aggregates logs from all nodes and
+projects into Elasticsearch and provides a Kibana UI for users to view
+logs that they have access to: for ordinary users, logs from any projects
+they are able to view, and for cluster-admin users, any logs at all.
+
```

Comment:
Again emphasizing that there is nothing per-project about this, and specifying what users should be able to see which I neglected to do anywhere else.